core/settings.py

```diff
@@ -54,9 +54,11 @@ class Common(Configuration):
     ]
 
     WSGI_APPLICATION = "core.wsgi.application"
-
-    default_db = "sqlite:///" + os.path.join(BASE_DIR, "db.sqlite3")
-    DATABASES = {"default": dj_database_url.config(default=default_db)}
+    os.environ.setdefault(
+        'DATABASE_URL',
```

Comment:
O `DATABASE_URL` padrão pode ser configurado no .env.example.
Entrando no flow de personalizações.

Reply:
Coloquei no docker-compose. Acho salutar deixar o sqllite por padrão.

Reply:
Devemos seguir, provavelmente, com o postgres tomando um papel obrigatório na etapa de desenvolvimento também. Dá uma olhada aqui: https://github.com/DadosAbertosDeFeira/maria-quiteria/issues/66

Reply:
Eu discordo. Nem sempre precisamos do postgres para toda e qualquer tarefa, como criar uma página estática. Mas não tenho opinões fortes sobre isso.

Reply:
Vamos precisar de _full text search_, como discutido na #66. O ORM do Django só implementa essas funções (sem _fallback_) para PostgreSQL, não? Teria alguma alternativa para resolver o discutido na #66 sem se amarrar ao Postgres?

Reply:
Acho que não. Fiz um teste sobre isso recente com sqlite e quebra mesmo, sem _fallback_ (a não ser que alguém tenha outra sugestão). Eu sei que não estava no escopo inicial do PR, @matheuscas, mas provavelmente vamos ter que organizar essas mudanças juntos (`.env`, postgres etc). :tulip:

Reply:
Como eu disse, apesar de discordar, tanto faz para mim. É só tirar o sqlite e se basear inteiramente no `DATABASE_URL `.

Reply:
@matheuscas, acho esse ponto importantíssimo e não queria ver um _apesar de discordar_. Queria entender o pq você discorda e como você contornaria o gargalo que o _full text search_ resolve sem depender do Postgres. Talvez a gente esteja perdendo algo e gostaria muito que tu falasse mais do pq tu discorda e como resolveria esse impasse ; )

Reply:
Não tem impasse, gente. Sério. 😄 Eu só **acho** que para DEV, que envolve um largo leque de atividades tanto simplórias quanto complexas, que não precisamos de um banco de dados. Mas como vamos adotar o docker, na média, isso tudo fica suavizado. Sigamos a reta. 😃  

Ps.: Minha experiência in full text search é só com ElasticSearch e/ou Algolia, então eu estou bem por fora do que vocês estou discutindo. 😅

Reply:
O problema é que temos que rodar as migrations pra poder usar o Django local e testar as coisas, tanto simplórias quanto complexas. Se usamos uma funcionalidade exclusiva do postgres, como o full text search ou lists, em qualquer model, não conseguimos aplicar as migrations usando um banco como o sqlite: simplesmente dá erro. Aí não dá pra rodar os testes, nem abrir o django admin, etc. Não é como se fosse opcional e estamos forçando um uso do banco.